templates/shared/event-list.tmpl

```diff
@@ -0,0 +1,30 @@
+{{ define "event-list" }}
+    <div class="event-list">
+
+        {{ range . }}
+            <div class="event" >
+                <a href="{{ .URL }}" class="event-box">
+                    <div class="event-figure">
+                            <img src="{{.ImageURL}}" class="event-img"/>
+                    </div>
+                    <div class="event-details">
+                            <h2 class="event-date">{{ .GetShortDate }}</h2>
+
+                            <h3 class="event-name">{{ .Name }}</h3>
+
+
+                            <p class="event-location"> {{.Location}}</p>
+                    </div>
+                    <div class="event-footer">
+                          <span class="event-attendees"> {{ .GetInterestLine }} </span>
+                          <span class="event-facebook-wrap">
+                            <img src="assets/images/social-media/facebook-grey.png" alt="Facebook" class="event-facebook">
+                          </span>
+                    </div>
+                </a>
+            </div>
+
+        {{end}}
+    </div>
+
+{{end}}
```

Comment:
As we are using 2 spaces tabs along our templates, It is a good idea to maintain consistency in tabs-length.